### ISSUE TITLE
--train_frac / --valid_frac

### ISSUE THREAD

**Issue by lowtronik** [crazydonkey200/tensorflow-char-rnn#18]
I've noticed that your default settings are  train_frac= 0.9  and   valid_frac=0.05 , is that on purpose?

**Comment by crazydonkey200** [comment]
Hi, that is just one common setup that uses first 90% as training set, next 5% as validation set and the last 5% as test set. You could change it to suit your needs.

**Comment by lowtronik** [comment]
oh, ok I was just wondering were the rest 0.05% was going

thanks
